apps/api/src/server/applications/application/documents/document.controller.js

```diff
@@ -489,6 +489,24 @@ export const storeDocumentVersion = async (request, response) => {
 		throw new BackOfficeAppError(`Document not found: guid ${guid}`, 404);
 	}
 
+	if (documentVersionMetadataBody.transcript) {
+		const transcriptReference = documentVersionMetadataBody.transcript;
+
+		const transcriptDocument = await documentRepository.getByReferenceRelatedToCaseId(
+			transcriptReference,
+			+caseId
```

Comment:
```suggestion
			Number(caseId)
```

Better to be explicit.